dask_sql/physical/rel/custom/analyze.py

```diff
@@ -54,7 +56,9 @@ def convert(
         statistics = statistics.append(
             pd.Series(
                 {
-                    col: str(python_to_sql_type(df[mapping(col)].dtype)).lower()
+                    col: str(python_to_sql_type(df[mapping(col)].dtype).getSqlType())
+                    .rpartition(".")[2]
+                    .lower()
```

Comment:
Is there any way to get the string name of a `SqlTypeName` variant? Right now this code is used in a lot of places to turn something like `datafusion.SqlTypeName.DOUBLE` into `'DOUBLE'` and am wondering if there's a cleaner alternative.

Reply:
we can probably override the `__str__` method that's autp generated by pyo3 in the bindings to just return the name instead.